generate_parameter_library_py/generate_parameter_library_py/jinja_templates/cpp/parameter_library_header

```diff
@@ -123,6 +123,17 @@ struct StackParams {
       return params_;
     }
 
+    bool try_get_params(Params & params_in) const{
```

Comment:
```suggestion
    bool try_get_params(Params & params_in) const {
```

Reply:
I was unaware of the "Suggested changes" feature, so I responded by committing directly to your suggestions.